decidim-proposals/app/cells/decidim/proposals/participatory_text_proposal_cell.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require "cell/partial"
+
+module Decidim
+  module Proposals
+    # This cell renders the participatory text proposal card for an instance of a Proposal
+    # the default size is the Medium Card (:m)
+    class ParticipatoryTextProposalCell < Decidim::ViewModel
+      include ProposalCellsHelper
+      include Cell::ViewModel::Partial
+      include Messaging::ConversationHelper
+
+      delegate :current_organization, to: :controller
+      delegate :user_signed_in?, to: :parent_controller
```

Comment:
Why do you need `user_signed_in?` instead of `current_user`?

Reply:
I guess the history is 
```
proposal_cell ==copy+paste==> collaborative_draft_cell ==copy+paste==> participatory_text_proposal_cell
```

But nobody uses that method...
I'll clean it up